diff --git a/bin/CrossMap.py b/bin/CrossMap.py
index ecc36a4..e29cf08 100755
--- a/bin/CrossMap.py
+++ b/bin/CrossMap.py
@@ -446,6 +446,8 @@ def crossmap_vcf_file(mapping, infile,outfile, liftoverfile, refgenome):
 				
 				fields[3] = refFasta.fetch(str(a[1][0]),a[1][1],a[1][2]).upper()
 				
+				if a[1][3] == '-':
+					fields[4] = _revcomp_dna(fields[4])
 				
 				if fields[3] != fields[4]:
 					print >>FILE_OUT, '\t'.join(map(str, fields))
@@ -462,6 +464,15 @@ def crossmap_vcf_file(mapping, infile,outfile, liftoverfile, refgenome):
 	printlog (["Failed to map:", str(fail)])
 				
 			
+def _revcomp_dna(seq):
+	'''
+	Take in a string sequence of DNA and reverse complement it
+	'''
+	complement = {'A': 'T', 'C': 'G', 'G': 'C', 'T': 'A', 'Y': 'R', 'R': 'Y', 'S': 'W', 'W': 'S', 'K': 'M', 'M': 'K', 'B': 'V', 'V': 'B', 'D': 'H', 'H': 'D', 'N': 'N' }
+	complemented_bases = reversed([complement.get(base,base) for base in list(seq)])
+	revcommp_seq = ''.join(complemented_bases)
+	return revcommp_seq
+
 def crossmap_bed_file(mapping, inbed,outfile=None):
 	'''
 	Convert genome coordinates (in bed format) between assemblies.
